dbt_subprojects/tokens/models/transfers_and_balances/arbitrum/_schema.yml

```diff
@@ -191,3 +191,26 @@ models:
       - name: symbol
       - name: decimals
       - name: collection_name
+
+  - name: tokens_arbitrum_net_transfers_daily
+    meta:
+      blockchain: arbitrum
+      sector: tokens
+      contributors: 0xRob, krishhh
+    config:
+      tags: ['tokens', 'net_transfers', 'daily']
+    description: "Total transfer amount per day"
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - block_date
+
+  - name: tokens_arbitrum_net_transfers_daily_address
+    meta:
+      blockchain: arbitrum
+      sector: tokens
+      contributors: 0xRob, krishhh
+    config:
+      tags: ['tokens', 'net_transfers']
+    description: "Total transfer amount per day per address"
```

Comment:
```suggestion
    description: "Total transfer amount per day per address"
    data_tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - blockchain
            - block_date
            - address
```

Reply:
rule of thumb: for incremental models, when we add a unique key in model config block, we should add corresponding test here in schema to ensure uniqueness

this will compile to a raw sql statement that checks for dupes by grouping on these columns

i may have forgotten to add this in my old PR, oops 🤷